src/lms-ds-installer/README.md

```diff
@@ -0,0 +1,4 @@
+# LMS-DS-Installer
+
+Scripts for installing the Ed-Fi Learning Management System Data Store (LMS-DS)
+sql schema script. Further details to follow when ready.

```

Comment:
Will you leave it like this for now?

Reply:
Would be helpful to redirect the users to the readme of src/lms-ds-loader when they get here

Reply:
I was going to leave like this for now, until [FIZZ-88](https://tracker.ed-fi.org/browse/FIZZ-88), which is the ticket for working on this installer. I can add a little more information right now though, for anyone who happens to pick up this source code.